app/controllers/admin/ticket_scannings_controller.rb

```diff
@@ -1,21 +1,16 @@
 module Admin
   class TicketScanningsController < Admin::BaseController
     before_action :authenticate_user!
-    before_action :find_physical_ticket_by_token
-    load_and_authorize_resource through: :physical_ticket, singleton: true, shallow: true
+    load_resource :physical_ticket, find_by: :token
+    # We authorize manually in these actions
+    skip_authorize_resource only: [:create]
```

Comment:
why is this needed? You are not authorizing the resource? I think you can remove lines 4 and 5 and everything will keep the same. :thinking: